bin/aapp_dr_runner.py

```diff
@@ -167,8 +162,8 @@ def cleanup_aapp_logfiles_archive(config):
     try:
         directory_list = glob(
             '%s/*' % config['aapp_processes'][config.process_name]['aapp_log_files_archive_dir'])
-        dummy = [delete_old_dirs(s, config['aapp_processes'][config.process_name][
-                                 'aapp_log_files_archive_length']) for s in directory_list if os.path.isdir(s)]
+        _ = [delete_old_dirs(s, config['aapp_processes'][config.process_name][
+            'aapp_log_files_archive_length']) for s in directory_list if os.path.isdir(s)]
```

Comment:
I don't think a list comprehension is indicated here.
```suggestion
        for s in directory_list:
            if os.path.isdir(s):
                delete_old_dirs(s, config['aapp_processes'][config.process_name]['aapp_log_files_archive_length'])
```